tests_integration/nativeapp/test_events.py

```diff
@@ -53,9 +53,10 @@ def test_app_events_mutually_exclusive_options(
             ["app", "events", *command],
             env=TEST_ENV,
         )
-        assert result.exit_code == 1, result.output
+
+        assert result.exit_code == 2, result.output
         assert (
-            f"{flag_names[0]} and {flag_names[1]} cannot be used together."
```

Comment:
IMHO "cannot be used together" message is more understandable, WDYT?

(I don't want to stop PR with this discussion - feel free to merge if you think otherwise)

Reply:
Yes it is less understandable, I change it to simultaneously instead of together, WDYT?
The full message is like the following:
`Parameters --file and --stdin are incompatible and cannot be used simultaneously.`